images/ubuntu/scripts/build/install-dotnetcore-sdk.sh

```diff
@@ -63,13 +63,15 @@ apt-get update
 sdks=()
 for version in ${dotnet_versions[@]}; do
     release_url="https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/${version}/releases.json"
-    releases=$(cat "$(download_with_retry "$release_url")")
-    if [[ $version == "6.0" ]]; then
-        sdks=("${sdks[@]}" $(echo "${releases}" | jq -r 'first(.releases[].sdks[]?.version | select(contains("preview") or contains("rc") | not))'))
-    else
-        sdks=("${sdks[@]}" $(echo "${releases}" | jq -r '.releases[].sdk.version | select(contains("preview") or contains("rc") | not)'))
-        sdks=("${sdks[@]}" $(echo "${releases}" | jq -r '.releases[].sdks[]?.version | select(contains("preview") or contains("rc") | not)'))
-    fi
+    releases=$(download_with_retry "$release_url")
+
+    # Extract SDK versions from the releases metadata
+    sdk_versions=($(echo "$releases" | jq -r '.releases[].sdk.version | select(contains("preview") or contains("rc") | not)'))
+    sdks+=("${sdk_versions[@]}")
+
+    # Extract additional SDK versions from the releases metadata
+    additional_sdks=($(echo "$releases" | jq -r '.releases[].sdks[]?.version | select(contains("preview") or contains("rc") | not)'))
+    sdks+=("${additional_sdks[@]}")
```

Comment:
Why do we change this?

Reply:
Earlier it is looking and downloading  for only dot net 6.0 , it throws error when building the pipeline, so i thought  to remove previous script snippet and added this to remove those boundaries.